lib/tools/updateArtistInfo.tsx

```diff
@@ -0,0 +1,36 @@
+import { z } from "zod";
+import { tool } from "ai";
+import { getSupabaseServerAdminClient } from "@/packages/supabase/src/clients/server-admin-client";
+import { ArtistToolResponse } from "@/types/Tool";
+
+const updateArtistInfo = (question: string) =>
+  tool({
+    description: `IMPORTANT: Always call this tool for ANY question related to the updating photo(image, avatar) of artist:
+    Do NOT attempt to answer questions on these topics without calling this tool first.
+
+    Example questions that MUST trigger this tool:
+    - "Update the image for my artists."
+    - "Update the avatar for my artists."
+    - "Update my artist photo."
+    - "Update the artist photo for [artist name.]"`,
+    parameters: z.object({
+      artist_name: z.string().optional().describe("Name of artist"),
+    }),
```

Comment:
why have you limited this?
- description to `photo` but (specific)
- function name to `artist. info` (general)